health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -1368,4 +1570,1030 @@
+          return t("ES_COMMON_NA");
+      }
+    },
+  },
+  CampaignsInboxConfig: {
+    preProcess: (data, additionalDetails) => {
+      data.body.ProjectStaff = {};
+      data.body.ProjectStaff.staffId = [Digit.UserService.getUser().info.uuid];
+      data.params.tenantId = Digit.ULBService.getCurrentTenantId();
+      data.params.limit = data.state.tableForm.limit;
+      data.params.offset = data.state.tableForm.offset;
+      delete data.body.ProjectStaff.campaignName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)